resource/ckb.toml

```diff
@@ -101,7 +101,7 @@ bootnode_mode = false
 support_protocols = ["Ping", "Discovery", "Identify", "Feeler", "DisconnectMessage", "Sync", "Relay", "Time", "Alert", "LightClient", "Filter"]
 
 # [network.sync.header_map]
-# memory_limit = "600MB"
+# memory_limit = "256MB"
```

Comment:
Changed the default memory limit to smaller one, it can keep about the same amount of header data in memory as before.